charts/k8s-watcher/templates/clusterrole.yaml

```diff
@@ -430,7 +430,7 @@ rules:
     verbs:     ["get", "watch", "list"]
 {{- end}}
 
-{{- if and .Values.watcher.resources.customReadAPIGroups gt (len .Values.watcher.resources.customReadAPIGroups) 0 }}
+{{- if and .Values.watcher.resources.customReadAPIGroups gt ((len .Values.watcher.resources.customReadAPIGroups) 0) }}
```

Comment:
the brackets are in the wrong place
`(gt len .Values.watcher.resources.customReadAPIGroups 0)`